semantic-tsx/src/Language/TSX.hs

```diff
@@ -0,0 +1,21 @@
+{-# OPTIONS_GHC -freduction-depth=0 #-}
```

Comment:
@robrix - OK to leave this in for now? Or...

Reply:
👍